docs/management.md

```diff
@@ -418,6 +446,66 @@ management.oauth_provider_url = https://my-web-portal
 With the previous settings, the management UI exposes the HTTP endpoint `/login` which accepts `content-type: application/x-www-form-urlencoded` and it expects the JWT token in the `access_token` form field. This is the endpoint where the Web portal will redirect users to the management UI.
 Additionally, RabbitMQ also accepts a JWT token in the HTTP `Authorization` header when the user lands on the management UI.
 
+With `sp_initiated` logon types, there is no need to configure the `oauth_provider_url` if `auth_oauth2.issuer` was set. However, for `idp_initiated` flows the `auth_oauth2.issuer` url may not necessarily be the url where to send users to authenticate. When this occurs, the `management.oauth_provider_url` overrides the `auth_oauth2.issuer` url.
+
+### Support multiple OAuth 2.0 resources {#support-multiple-resources}
+
+RabbitMQ 3.13 introduced support for multiple OAuth 2.0 resources in the OAuth 2.0 plugin and in the Management plugin.
+
+Once you have configured the [OAuth 2.0 plugin](./oauth2#multiple-resource-servers) with all the required OAuth 2.0 resources, they are automatically supported by the management UI.
+
+Given the following OAuth 2.0 plugin configuration which consists of four OAuth 2.0 resources:
+```ini
+auth_oauth2.issuer = http://some_idp_url
+auth_oauth2.scope_prefix = rabbitmq.
+auth_oauth2.resource_servers.1.id = rabbit_prod
+auth_oauth2.resource_servers.2.id = rabbit_dev
+auth_oauth2.resource_servers.3.id = rabbit_qa
+auth_oauth2.resource_servers.4.id = rabbit_internal
+```
+
+This is the minimum configuration to enable all resources in the management UI:
+```ini
+management.oauth_enabled = true
+management.oauth_scopes = openid rabbitmq.tag:management rabbitmq.read:*/*
+```
+
+However, it is possible to configure the client_id, the labels for each resource, or the login type, the scopes
+for each resource or the provider_url. Or even disable some resources in the management UI.
```

Comment:
How about rewriting to: If you want to complete further configuration, refer to the following example, it shows how to:
  - configure the client_id
  - Add a label to each resource
  - Specify a login type
  - Add scopes for each resource 
   - Include the oauth_provider_url
   - deactivate some resources in the Management UI